kahuna/conf/kahuna.conf

```diff
@@ -1,28 +0,0 @@
-env APP=kahuna
-
-env USER=media-service
-env USER_HOME=/home/media-service
-env JAR=/home/media-service/app.jar
-
-env JAVA_OPTS="-Dfile.encoding=UTF-8 -Xmx300m -XX:MaxPermSize=64m -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
-
-env LOGFILE=/home/media-service/logs/stdout.log
-
-start on runlevel [2345]
-stop on runlevel [016]
-
-# NB: setuid is only supported on recent versions of upstart - i.e.
-#  on ubuntu not on amazon linux
-
-setuid media-service
-
-chdir /home/media-service
-
-# automatically restart if the process dies
-# respawn
-
-script
-  CMD="$USER_HOME/bin/$APP"
-  echo "$CMD" > $USER_HOME/logs/cmd.txt
-  $CMD > $LOGFILE 2>&1
```

Comment:
i don't understand what this script at the bottom was meant to do.... basically log that the app was run?